Source/Bogus/DataSet.cs

```diff
@@ -8,7 +8,7 @@ namespace Bogus
    /// <summary>
    /// Data set methods that access the BSON database of locales.
    /// </summary>
-   public class DataSet : ILocaleAware, IHasRandomizer
+   public class DataSet : ILocaleAware, IHasRandomizer, IDataset
```

Comment:
Cosmetic: `IDataset` should probably be named `IDataSet` to be consistent with the `DataSet` class.

Reply:
Yep, missed it, thanks